### ISSUE TITLE
After Upgraded to React Native 0.68.0 application not build

### ISSUE THREAD

**Issue by manimaran-aptean** [Scandit/barcodescanner-sdk-react-native#240]
Showing error in 

```
> Task :scandit-react-native-datacapture-core:compileDebugKotlin FAILED
e: /Users/xx/xx/xxx/node_modules/scandit-react-native-datacapture-core/android/src/main/kotlin/com/scandit/datacapture/reactnative/core/utils/TestThreads.kt: (29, 18): Return type of 'runOnQueue' is not a subtype of the return type of the overridden member 'public abstract fun runOnQueue(p0: Runnable!): Boolean defined in com.facebook.react.bridge.queue.MessageQueueThread'

```

**Comment by ScanditSupport** [comment]
Hi @manimaran-aptean,

Please accept our apologies for the issue, it is caused by this change:
https://github.com/facebook/react-native/commit/89faf0c9a87f6de68ca416d10566dbcbe80d9450

We're actively looking into this and will update the SDK to support React-Native 0.68.0 in the upcoming releases, thank you for your patience.

In case you have more detailed questions, please feel free to reach out to us directly: [Submit a request – Frequently Asked Questions (FAQ)](https://support.scandit.com/hc/en-us/requests/new)

**Comment by ScanditSupport** [comment]
Hi @manimaran-aptean,

I wanted to let you know that 6.13 SDK has been released and it includes the fix for React-Native 0.68.

In case you have any additional questions, please feel free to reach out to us directly: [Submit a request – Frequently Asked Questions (FAQ)](https://support.scandit.com/hc/en-us/requests/new)
